src/test/java/net/sf/jabref/importer/fileformat/SilverPlatterImporterTest.java

```diff
@@ -0,0 +1,85 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.BibtexEntryTypes;
+
+public class SilverPlatterImporterTest {
+
+    private SilverPlatterImporter testImporter;
+
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        testImporter = new SilverPlatterImporter();
+    }
+
+    @Test
+    public final void testIsRecognizedFormat() throws Exception {
+        try (InputStream stream = SilverPlatterImporterTest.class
+                .getResourceAsStream("SilverPlatterImporterTest1.txt")) {
+            Assert.assertTrue(testImporter.isRecognizedFormat(stream));
+        }
+    }
+
+    @Test
+    public final void testIsNotRecognizedFormat() throws Exception {
+        List<String> notAccepted = Arrays.asList("emptyFile.xml", "IsiImporterTest1.isi",
+                "InspecSilverPlatterImporterTest.txt", "oai2.xml", "RisImporterTest1.ris");
+        for (String s : notAccepted) {
+            try (InputStream stream = SilverPlatterImporterTest.class.getResourceAsStream(s)) {
+                Assert.assertFalse(testImporter.isRecognizedFormat(stream));
+            }
+        }
+    }
+
+    @Test
+    public final void testImportEntries1() throws Exception {
+
+        try (InputStream in = SilverPlatterImporter.class.getResourceAsStream("SilverPlatterImporterTest1.txt");
+                InputStream bibIn = SilverPlatterImporter.class.getResourceAsStream("SilverPlatterImporterTest1.bib")) {
+            List<BibEntry> entries = testImporter.importEntries(in, new OutputPrinterToNull());
+            Assert.assertEquals(1, entries.size());
+            BibtexEntryAssert.assertEquals(bibIn, entries.get(0));
```

Comment:
Is there an overload which accepts `entries` as a parameter instead of a single entry? If yes, then use it. Otherwise create it :)